### DISCUSSION TITLE
Does the i18n example (32_i18n) work?

### DISCUSSION THREAD

**Discussion by w3sly** [Kitware/trame#432] [General]
Hello

I have been looking at the internationalization example that you have posted in the repository (here: https://github.com/Kitware/trame/blob/master/examples/validation/core/32_i18n.py) of trame but, or I am not understanding it well , or does not work properly.

I understand that the idea is that the vuetify.VSelect() label has to change based on the selected language, but it doesn't work for me. I understand that the value of the variable $vuetify.lang.t('title') should return the title based on the language that is selected, but I don't see that working either. Do I misunderstand the idea of the example or is it not working properly? Is it necessary to use a callback to update the selector tag?

I added the following lines, below line 57:
```
toolbar.add_child("{{ $vuetify.lang.t('title') }}")
toolbar.add_child("{{ $vuetify.lang.t('selector') }}")
```

but, as the following images show, the text was not updated:
![Captura de pantalla de 2024-02-02 09-26-33](https://github.com/Kitware/trame/assets/95097694/a3a9681b-3e11-41cd-ba46-7c400f1fcc0c)
![Captura de pantalla de 2024-02-02 09-26-42](https://github.com/Kitware/trame/assets/95097694/2f6328a9-7c46-4dcd-8ea7-5ff929298f69)

Thanks in advance.

**Comment by jourdain** [comment]
No it is not working but it was some initial exploration as I have now idea how internationalization works with vuetify.
Maybe it might be easier to leverage with vuetify3?
